lib/Browser.js

```diff
@@ -22,12 +22,14 @@ class Browser extends EventEmitter {
   /**
    * @param {!Puppeteer.Connection} connection
    * @param {!Object=} options
+   * @param {?Puppeteer.ChildProcess} process
```

Comment:
Shouldn’t this be NodeJS.ChildProcess?

Reply:
For some reason, that's what we have in externs

Reply:
Ok I trust whatever I was doing to make it compile then